presentation/src/main/java/org/sagebionetworks/research/presentation/perform_task/PerformTaskViewModel.java

```diff
@@ -254,8 +254,6 @@ void taskInitFail(Throwable t) {
 
     @VisibleForTesting
     void taskInitSuccess() {
-        checkState(taskResult != null, "taskResultBuilder must be set before taskInitSuccess is called");
```

Comment:
Is this no longer a precondition?

Reply:
The TaskResult is no longer an autovalue class so it doesn't have a builder. As a result this error doesn't make sense anymore. I am not quite sure what to do in place of the error. I think a good approach might be to check if that task result is null and if it is initialize a new one. Thoughts?